content/data/people.yaml

```diff
@@ -52,3 +52,12 @@
   github: willunderwood
   gifPlaceHolder: ../../content/assets/people/will-gif-placeholder.jpg
   profilePictureGif: ../../content/assets/people/will.gif
+  - id: lyudmila
+  fullName: Lyudmila Lugovskaya
+  title: Data Scientist
+  bio: PhD in Finance, University of Cambridge, awarded scholarships. MSc in Psychology. Cross-industry experience of delivering practical data projects. Led data science teams. London/Barcelona based. 
```

Comment:
There is a white space at the end of this that can be deleted.

Reply:
423d0c82810d83cb42976d80074648f523105a77